src/main/webapp/app/exam/participate/exam-participation.component.html

```diff
@@ -56,22 +56,40 @@
                                         <div [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
                                             @switch (exercise.type) {
                                                 @case (QUIZ) {
-                                                    <jhi-quiz-submission-exam
-                                                        [quizConfiguration]="exercise"
-                                                        [studentSubmission]="exercise.studentParticipations[0].submissions![0]"
-                                                    />
+                                                    @if (exercise.studentParticipations[0].submissions) {
```

Comment:
to make sure that submissions[0] doesn't try to access an empty list, I think you should extend the check here.
Js evaluates `[]` to true

Reply:
Thanks for your suggestion. The idea here is only to eliminate non-null assertion operator (!). At the beginning of the exam, each exercise type except programming has initially the submissions length 1, therefore it should not access to an empty list. If we add that additional assertion here and submissions is an empty list somehow, we can't access that exercise during the exam, which should not be the case. (See screenshots) We still need to be able to access it if submissions is an empty list. 
![Group 157](https://github.com/user-attachments/assets/efb1fabd-24f0-402d-9894-8395dbf37d6c)